services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/dataaccess/SubmissionManagerImpl.java

```diff
@@ -282,7 +282,7 @@ public Submission updateStatus(UserInfo userInfo, SubmissionStateChangeRequest r
 			createApprovalsForSubmission(submission, modifiedBy, expiredOn, approvalsToCreateOrUpdate, accessorsToRevoke);
 
 			if (!accessorsToRevoke.isEmpty()) {
-				accessAprovalManager.revokeGroup(userInfo, submission.getAccessRequirementId(), submission.getSubmittedBy(), accessorsToRevoke);
+				accessAprovalManager.revokeGroupUnchecked(userInfo, submission.getAccessRequirementId(), submission.getSubmittedBy(), accessorsToRevoke);
```

Comment:
Note to self, authorizationManager.canReviewAccessRequirementSubmissions() is called above